sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -1942,10 +1963,6 @@ void captureSessionToken(RxDocumentServiceRequest request, RxDocumentServiceResp
 
                             });
 
-                        Set<PartitionKeyRange> partitionKeyRanges = partitionRangeItemKeyMap.keySet();
-                        List<PartitionKeyRange> ranges = new ArrayList<>();
-                        ranges.addAll(partitionKeyRanges);
```

Comment:
was this unused (dead code)?

Reply:
Correct - the partitionKeyRanges was/is never used